spec/features/tag_facets_to_a_page_spec.rb

```diff
@@ -270,7 +270,7 @@ def and_i_edit_facets_for_the_page(path)
   end
 
   def and_i_see_the_facet_values_form_prefilled_with(option)
-    facet_values_options = all('#facets_tagging_update_form_facet_values option').map(&:text)
+    facet_values_options = all('#facets_tagging_update_form_example_facet option').map(&:text)
```

Comment:
Nitpick: should this be `facets_tagging_update_form_example_facet_group`?

Reply:
It's not a facet group. It's the text from the options on a set of facet values.

Reply:
Ok, found it: https://github.com/alphagov/content-tagger/blob/ed1797613ffd42980b12cd26e3820e1489e76253/spec/support/publishing_api_helper.rb#L244

There are a lot of things named `example_facet....` so it took a while to find.